contrib/llvm-project-cmake/CMakeLists.txt

```diff
@@ -58,18 +56,25 @@ set (REQUIRED_LLVM_LIBRARIES
     LLVMDemangle
 )
 
-# if (ARCH_AMD64)
+if (ARCH_AMD64)
```

Comment:
Should we add RISC-V, PPC, S390X here, or disable JIT for them?

Reply:
I will add all architectures here.